### ISSUE TITLE
[ImageView] adding contrast/brightness controls to image view

### ISSUE THREAD

**Issue by larkin** [nasa/openmct#1324]
@VWoeltjen working on.

**Comment by VWoeltjen** [comment]
Filed #1326 with work in progress here. Needs markup attention from @charlesh88 before continuing. Reassign to me after that

**Comment by charlesh88** [comment]
@VWoeltjen I've committed in all my stuff. The component is implemented as a local control floating over the image, here's how it should look:

<img width="687" alt="screen shot 2016-11-15 at 12 30 52 pm" src="https://cloud.githubusercontent.com/assets/1056412/20322854/656126a0-ab2f-11e6-94b1-bc6e5ad91d74.png">

I changed the range max to 500 from 200 to allow more dynamic range to the image manipulation. I tried to find a way to symmetrically extend the bottom range, but that couldn't find a way to do that. I also added a reset button and updated the sample images to use imagery that stress tests the controls better. All the imagery is licensed for open reuse.
